src/sentry/tasks/sentry_apps.py

```diff
@@ -25,7 +25,7 @@
 )
 from sentry.models.sentryapp import VALID_EVENTS
 
-logger = logging.Logger("sentry.tasks.sentry_apps")
+logger = logging.getLogger("sentry.tasks.sentry_apps")
```

Comment:
@MeredithAnya This is pretty bad...I'm surprised this didn't just error out and die.

Reply:
it logged out `No handlers could be found for logger "sentry.tasks.sentry_apps"`... lol